Ladybird/BrowserWindow.cpp

```diff
@@ -54,6 +54,13 @@ BrowserWindow::BrowserWindow(Browser::CookieJar& cookie_jar, StringView webdrive
     close_current_tab_action->setShortcuts(QKeySequence::keyBindings(QKeySequence::StandardKey::Close));
     menu->addAction(close_current_tab_action);
 
+    auto* open_file_action = new QAction("&Open File...");
```

Comment:
we should pass the BrowserWindow instance `this` as the parent object to this QAction to make sure we don't leak it on process exit.

```suggestion
    auto* open_file_action = new QAction("&Open File...", this);
```